util/dvsim/Deploy.py

```diff
@@ -882,7 +899,7 @@ def _test_passed(self):
                                     colalign=colalign)
 
         # Delete the cov report - not needed.
-        os.system("rm -rf " + self.log)
+        os.remove(self.log)
```

Comment:
This doesn't have quite the same semantics as it had: I think you probably don't want an error if `self.log` didn't happen to exist before. If you write the helper function suggested above, you could just use that.